src/components/views/voip/CallView.tsx

```diff
@@ -566,7 +617,33 @@ export default class CallView extends React.Component<IProps, IState> {
                         />
                     </div>
                 </div>
-                {holdTransferContent}
+                <div className="mx_CallView_holdTransferContent">{_t("Connecting")}</div>
```

Comment:
connecting states is a thing we want to add and have designs for: do you have a screenshot of this?

Reply:
![Screenshot_20210416_111214](https://user-images.githubusercontent.com/25768714/115002072-9e254580-9ea4-11eb-9d9d-589ad992cc14.png)

This is the current boring thing. I am happy to make this to match the design if someone sent me a link to the Figma file

Reply:
Actually the designs have a few more changes we would want to do separately anyway so this looks 👌